astacus/coordinator/list.py

```diff
@@ -4,23 +4,14 @@
 
 """
 from astacus.common import ipc, magic
-from astacus.common.json_view import (
```

Comment:
Lets remove `json_view` since we no longer work with json directly.

Reduce the amount of code we need to maintain.

Reply:
Yes, I want to do that in a separate PR to avoid huge line change count.